exercises/largest-series-product/example.rs

```diff
@@ -0,0 +1,24 @@
+use std::str::FromStr;
+
+pub fn lsp(string_digits: &str, span: usize) -> Result<u64, String> {
+    if span == 0 {
+        return Ok(1);
+    }
+
+    if string_digits.chars().any(|c| !c.is_numeric()) {
+        return Err(String::from("All characters must be numbers"));
+    }
+
+    let products: Vec<u64> = string_digits.chars()
+        .map(|c| u64::from_str(&c.to_string()).unwrap())
```

Comment:
This will allocate a string on the heap just to throw it away. A more memory friendly version would be:

`.map(|c| c.to_digit(10).unwrap())`